styles/styles.css

```diff
@@ -0,0 +1,104 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  background-color: #3e3c3c;
+  align-items: center;
+  padding: 0 2rem;
+  color: rgb(255, 255, 255);
+  width: 100vw;
+  height: 92px;
+}
+
+.logo {
+  font-family: "Roboto", sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  text-align: center;
+  text-decoration: none;
+  color: white;
+}
+
+/* div ul {
+  display: none;
+}
+
+li {
+  list-style: none;
+}
+
+ul li a {
+  text-decoration: none;
+  padding: 10px;
+}
```

Comment:
- [ ] Kindly remove all commented-out codes. Removing commented-out codes makes your work look very clean and professional. 👍 💻